dome.gd

```diff
@@ -69,6 +75,8 @@ func set_sprite(path: String):
 # Only called when becomes fully infested
 func _on_fully_infested():
 	$DomeLostCountdownTimer.start(INFESTATION_COUNTDOWN)
+	##TODO: use signal in DomeConnections instead?
+	DomeConnections.dome_start_spread(self as Variant as Area2D)
```

Comment:
this is how I fixed the `self` static typing issue. I just cast it to the Node I expect, then I get to keep the static typing in other places